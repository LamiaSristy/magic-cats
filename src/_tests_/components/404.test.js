import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom/extend-expect';
import FourOFour from '../../components/404';

const setup = () => shallow(<FourOFour />);

describe('404 component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });

  it('Should render', () => {
    const element = component.find('.four-o-four');
    expect(element.length).toEqual(1);
  });

  test('renders the header text of the Home page', () => {
    const header = component.find('.fof-header');
    expect(header).toMatchSnapshot();
  });

  test('renders the pagenotfound the page', () => {
    const pagenotfound = component.find('.fof-not-found');
    expect(pagenotfound).toMatchSnapshot();
  });
});
