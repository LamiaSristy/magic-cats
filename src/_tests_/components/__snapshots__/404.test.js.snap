// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 component renders the header text of the Home page 1`] = `ShallowWrapper {}`;

exports[`404 component renders the pagenotfound the page 1`] = `ShallowWrapper {}`;

exports[`404 component renders the title text of the Home page 1`] = `ShallowWrapper {}`;

exports[`404 component renders the title text of the Home page 2`] = `ShallowWrapper {}`;
